test/input_data/config.edn

```diff
@@ -0,0 +1,48 @@
+{
+ ;; Section 1: Landscape data to be shared by all simulations
+ :db-spec                     {:classname   "org.postgresql.Driver"
+                               :subprotocol "postgresql"
+                               :subname     "//localhost:5432/gridfire"
+                               :user        "gjohnson"}
```

Comment:
should the user field be postgres?

Reply:
it could be, although there is a conceptual separation between the `postgres` user that has access to all of postgres and the user (here, the name of the upstream author as in `resources/sample_config.edn`) with access to the gridfire database.

i'll add some comments on config2 to clarify.

Reply:
i've also added an `:username` optarg to `startup-health-check` to change the username via [`with-redefs`](https://www.braveclojure.com/zombie-metaphysics/)